V0_Display/Documentation/Setup_and_Flashing_Guide.md

```diff
@@ -6,7 +6,9 @@
 
 	a) If you have a pre-flashed board:
 		-Install the boot jumper and reset the board. This will put the board into DFU mode.
-	
+		
+		This is required for the LDO Display
+		
```

Comment:
this is not unique to the LDO displays Best to ASSUME the board is already flashed and explicitly tell the user to install the jumper.